content/en/docs/tasks/configure-pod-container/resize-container-resources.md

```diff
@@ -173,6 +173,8 @@ spec:
 Let's say the CPU requirements have increased, and 0.8 CPU is now desired. This
 is typically determined, and may be programmatically applied, by an entity such as
 [VerticalPodAutoscaler](https://github.com/kubernetes/autoscaler/tree/master/vertical-pod-autoscaler#readme) (VPA).
+The 'InPlacePodVerticalScaling' [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) must be
+enabled to scale Pod resources without restarting the Pod.
```

Comment:
```suggestion
Kubernetes has experimental support for changing the resource allocation for a running
Pod; the alternative is to delete the new Pod and let a
[workload controller](/docs/concepts/workloads/controllers/) make a replacement Pod
that has a different size.
To use that experimental support, you must enable the
`InPlacePodVerticalScaling` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/) for your control plane and for all nodes in your cluster.
```

However, this paragraph really belongs in the prerequisites section, earlier in the page.